arm-apimanagement/2016-07-07/swagger/apimmanagement.json

```diff
@@ -6321,6 +6574,73 @@
         "type"
       ],
       "description": "Parameters supplied to the Update Logger operation."
+    },    
+    "BackendGetContract": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Uniquely identifies the backend within the current API Management service instance. The value is a valid relative URL in the format of /backends/{backendId} where {backendId} is a backend identifier.",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackendParameters"
+        }
+      ],
+      "required": [
+        "host",
+        "skipCertificateChainValidation"
+      ],
+      "description": "Backend."
+    },
+    "BackendPaged": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BackendGetContract"
+          },
+          "description": "Backend values."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Paged Backend list representation."
+    },
+    "BackendCreateParameters": {      
+      "allOf": [
+          {
+            "$ref" : "#/definitions/BackendParameters"
+          }
+       ],
+      "required": [
+        "host"
```

Comment:
This would be really useful. Here I just had to define one Base Class and use it while creating Get and Create Contract. Now I will have to expand the Create Contract also to associate the required properties.